src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -7407,6 +7407,24 @@ HRESULT DacDbiInterfaceImpl::GetReJitInfo(VMPTR_MethodDesc vmMethod, CORDB_ADDRE
     return S_OK;
 }
 
+HRESULT DacDbiInterfaceImpl::AreOptimizationsDisabled(VMPTR_Module vmModule, mdMethodDef methodTk, OUT BOOL* pOptimizationsDisabled)
+{
+    DD_ENTER_MAY_THROW;
+    PTR_Module pModule = vmModule.GetDacPtr();
+    if (pModule == NULL || pOptimizationsDisabled == NULL || TypeFromToken(methodTk) != mdtMethodDef)
+    {
+        return E_INVALIDARG;
+    }
+    {
+        CodeVersionManager::LockHolder codeVersioningLockHolder;
```

Comment:
These should follow suit of other methods here and guard on FEATURE_REJIT (just for consistency, as some point we might want to remove). Also, the lock is not necessary in the DAC

Reply:
Let me know if my latest commit is what you meant.

Reply:
I tested the latest commit, it works!